app/models/form/lettings/pages/property_local_authority.rb

```diff
@@ -3,7 +3,8 @@ def initialize(id, hsh, subsection)
     super
     @id = "property_local_authority"
     @depends_on = [
-      { "is_la_inferred" => false, "is_general_needs?" => true, "address_search_given?" => true },
+      { "is_la_inferred" => false, "is_general_needs?" => true, "form.start_year_after_2024?" => false },
+      { "is_la_inferred" => false, "is_general_needs?" => true, "form.start_year_after_2024?" => true, "address_search_given?" => true },
```

Comment:
I think address_search_given is only applicable to 2024 onwards, cause we're not doing address searches before then. This kinda meant that we haven't routed to LA question for 3 months, so will need to infer things where they can be inferred and set logs in progress in other places.

Reply:
That's not good!

Do we need to recalculate status on some logs then?

Reply:
I'm hoping we can infer the LAs on most logs, otherwise, year, would need to recalculate

Reply:
Cool - do we need anything to trigger that?

Reply:
I was gonna do it manually, but that was probably a bad idea. There's a rake task now